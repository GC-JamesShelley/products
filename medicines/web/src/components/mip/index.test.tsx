import { shallow } from 'enzyme';
import React from 'react';
import Mip from './index';

jest.mock('next/router', () => ({
  useRouter: () => ({
    query: {
      searchTerm: 'drug',
      page: 1,
      substance: 'whisky',
      disclaimer: true,
    },
  }),
}));

describe(Mip, () => {
  it('should render', () => {
    const component = shallow(<Mip />);
    expect(component).toMatchSnapshot();
  });
});
