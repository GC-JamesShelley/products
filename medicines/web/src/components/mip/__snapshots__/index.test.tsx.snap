// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mip should render 1`] = `
<styled.div>
  <section
    className="search"
  >
    <Search
      onSearchBlur={[Function]}
      onSearchChange={[Function]}
      onSearchSubmit={[Function]}
      search=""
    />
    <DrugIndex
      horizontal={true}
      items={
        Array [
          Object {
            "name": "A",
          },
          Object {
            "name": "B",
          },
          Object {
            "name": "C",
          },
          Object {
            "name": "D",
          },
          Object {
            "name": "E",
          },
          Object {
            "name": "F",
          },
          Object {
            "name": "G",
          },
          Object {
            "name": "H",
          },
          Object {
            "name": "I",
          },
          Object {
            "name": "J",
          },
          Object {
            "name": "K",
          },
          Object {
            "name": "L",
          },
          Object {
            "name": "M",
          },
          Object {
            "name": "N",
          },
          Object {
            "name": "O",
          },
          Object {
            "name": "P",
          },
          Object {
            "name": "Q",
          },
          Object {
            "name": "R",
          },
          Object {
            "name": "S",
          },
          Object {
            "name": "T",
          },
          Object {
            "name": "U",
          },
          Object {
            "name": "V",
          },
          Object {
            "name": "W",
          },
          Object {
            "name": "X",
          },
          Object {
            "name": "Y",
          },
          Object {
            "name": "Z",
          },
          Object {
            "name": "0",
          },
          Object {
            "name": "1",
          },
          Object {
            "name": "2",
          },
          Object {
            "name": "3",
          },
          Object {
            "name": "4",
          },
          Object {
            "name": "5",
          },
          Object {
            "name": "6",
          },
          Object {
            "name": "7",
          },
          Object {
            "name": "8",
          },
          Object {
            "name": "9",
          },
        ]
      }
      title="or find by active substance:"
    />
  </section>
  <YellowCard />
  <MipText />
</styled.div>
`;
